src/panels/lovelace/cards/hui-row-card.js

```diff
@@ -0,0 +1,73 @@
+import createCardElement from '../common/create-card-element.js';
+import createErrorConfig from '../common/create-error-card-config.js';
+
+class HuiRowCard extends HTMLElement {
+  constructor() {
+    super();
+    this.attachShadow({ mode: 'open' });
+    this._elements = [];
+  }
+
+  getCardSize() {
+    let size = 1;
+    this._elements.forEach((element) => {
+      if (typeof element.getCardSize === 'function') {
```

Comment:
We're having this logic at many places, probably consider extracting into function